handler/handler_test.go

```diff
@@ -20,6 +26,33 @@ import (
 )
 
 func TestNew(t *testing.T) {
+	rt := &role.Token{
```

Comment:
maybe it is better to just mock the interface than use the struct directly?

something like this:
https://github.com/yahoojapan/authorization-proxy/blob/c80098034a01d3b0ffc3c51736b6da624101b3b4/handler/roundtripper_mock.go

Reply:
Fixed
https://github.com/yahoojapan/authorization-proxy/pull/66/commits/72b90da34685ad95fb5ba452f94fd4ed26cb88a2